### PR TITLE
{bio}[foss/2020b] CIRI-long v1.0.2 w/ Python 3.8.6

### PR DESCRIPTION
**Pull request by zarybnicky**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 4 out of 4 (1 easyconfigs in total)
nuc.lan - Linux Fedora 33, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.9.6
See https://gist.github.com/d900eac83c35ce1c6c1f380aba7da4dc for a full test report.

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 5 out of 5 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/8b5e0cd886a31f4dce67d8a4c82bda22 for a full test report.

File: easybuild/easyconfigs/c/CIRI-long/CIRI-long-1.0.2-foss-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        'checksums': ['ed1460efc9c6785df28065b7e93e93c92227f623a181f1a852dca6e6acb1a15f'],
+    }),
+    ('python-Levenshtein', '0.12.1', {
+        'modulename': 'Levenshtein',
+        'checksums': ['554e273a88060d177e7b3c1e6ea9158dde11563bfae8f7f661f73f47e5ff0911'],
+    }),
+    ('bwapy', '0.1.4', {
+        'checksums': ['fcdad0c7311753d550b3a3aefbc2619380e7e35e7aeaa386fd491cba47e71824'],
+    }),
+    ('nose', '1.3.7', {
```
Comment (boegel) [inline easybuild/easyconfigs/c/CIRI-long/CIRI-long-1.0.2-foss-2020b.eb:36]: @zarybnicky This is already included with `Python` itself, so can be removed here.

**Review by verdurin — APPROVED** [review]
Looks fine.

File: easybuild/easyconfigs/c/CIRI-long/CIRI-long-1.0.2-foss-2020b.eb [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    ('pyspoa', '0.0.8'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('mappy', '2.17', {
+        'checksums': ['ed1460efc9c6785df28065b7e93e93c92227f623a181f1a852dca6e6acb1a15f'],
+    }),
+    ('python-Levenshtein', '0.12.1', {
```
Comment (boegel) [inline easybuild/easyconfigs/c/CIRI-long/CIRI-long-1.0.2-foss-2020b.eb:29]: @zarybnicky We have dedicated easyconfigs for this, we should keep that up?

**Comment by verdurin** [comment]
Going in, thanks @zarybnicky!

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/c/CIRI-long/CIRI-long-1.0.2-foss-2020b.eb (hunk 1 above)
Comment (zarybnicky) [reply easybuild/easyconfigs/c/CIRI-long/CIRI-long-1.0.2-foss-2020b.eb:36]: I'll submit an update PR

**Comment by boegel** [comment]
follow-up PR to fix remarks: #13985

File: easybuild/easyconfigs/c/CIRI-long/CIRI-long-1.0.2-foss-2020b.eb (hunk 2 above)
Comment (zarybnicky) [reply easybuild/easyconfigs/c/CIRI-long/CIRI-long-1.0.2-foss-2020b.eb:29]: I looked for all deps other than python-Levenshtein :) I'll submit an update PR

**Pull request merged**